apps/dsb-client-gateway-api/src/app/types/environment-variables.ts

```diff
@@ -120,6 +120,34 @@ export class EnvironmentVariables {
   @ValidateIf(EnvironmentVariables.isAWSSecretsManagerEnabled)
   AWS_SECRET_PREFIX = '/ddhub/';
 
+  @IsString()
+  @ValidateIf(EnvironmentVariables.isAWSSecretsManagerEnabled)
+  AWS_ACCESS_KEY_ID: string;
+
+  @IsString()
+  @ValidateIf(EnvironmentVariables.isAWSSecretsManagerEnabled)
+  AWS_SECRET_ACCESS_KEY: string;
+
+  @IsString()
+  @ValidateIf(EnvironmentVariables.isAzureKeyVaultEnabled)
+  AZURE_VAULT_NAME: string;
+
+  @IsString()
+  @ValidateIf(EnvironmentVariables.isAzureKeyVaultEnabled)
+  AZURE_SECRET_PREFIX = 'ddhub/';
```

Comment:
Why can't this have leading slash same as `AWS_SECRET_PREFIX`?

Reply:
Azure only allows `A-Z` `a-z` `0-9` and `-`. See my comment above and we can have a single `SECRET_PREFIX` if you agree